src/arm/bbai-bone-buses.dtsi

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2020 Deepak Khatri <deepaklorkhatri7@gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include <dt-bindings/board/am572x-bbai-pins.h>
+
+&dra7_pmx_core {
+	bone_uart_1_pins: pinmux_bone_uart_1_pins {
+		pinctrl-single,pins = <
+			P9_24( PIN_OUTPUT_PULLUP | MUX_MODE3)	/* gpio6_15.uart10_txd */
```

Comment:
Why do we need pull-up/pull-downs in UART mode?

Reply:
zmatt told me that floating pins cause hardware degradation (I totally agree with him) and then I looked online and found out that we do use pull-ups for the UART pins in practice. So I included these.